src/modules/auth/auth.controller.ts

```diff
@@ -162,4 +165,93 @@ export class AuthController extends Controller {
       data: { message: "Forgot password email send successfully" },
     });
   }
+
+  @Post("/resetPassword")
+  @Middlewares(validateData(authValidations.resetPass))
+  public async resetPassword(
+    @Body() body: AuthResetPass
+  ): Promise<APIResponse<Message>> {
+    const { password, ...restOfBody } = body;
+
+    const verified = await otpService.verify(restOfBody);
+    if (!verified) {
+      this.setStatus(statusConst.unAuthenticated.code);
+      return toResponse({ error: statusConst.unAuthenticated.message });
+    }
+
+    const newPassword = await authSerivce.hashPassword(password);
+
+    await prisma.users.update({
+      where: { email: body.email },
+      data: { password_hash: newPassword },
+    });
+
+    return toResponse({
+      data: { message: "Password has been reset successfully" },
+    });
+  }
+
+  @Post("/changePassword")
+  @Middlewares(validateData(authValidations.changePass))
+  public async changePassword(
+    @Body() body: AuthChangePass
+  ): Promise<APIResponse<Message>> {
+    const { email, password } = body;
+
+    const newHashedPassword = await authSerivce.hashPassword(password);
+
+    await prisma.users.update({
+      where: {
+        email: email,
```

Comment:
user should be targeted through the id instead of email. Similar to how it's done in `signoutAll` function